### DISCUSSION TITLE
Filter images by label in ImagePolicy

### DISCUSSION THREAD

**Discussion by tibberg** [fluxcd/flux2#903] [Proposals] [answered]
Is it possible to filter the images by label on ImagePolicy? 
It would be great if we could restrict the images by docker labels.

**Comment by stefanprodan** [comment, marked as answer]
No this is not possible, flux2 doesn't download any image layers (the labels are not exposed by OCI API) because we'll hit rate limits and create fake image download stats.

**Reply by tibberg** [reply]
Ah. I see. Thank you.
